common/app/views/fragments/header.scala.html

```diff
@@ -83,7 +83,7 @@
                             @fragments.inlineSvg("marque-36", "icon", List("rounded-icon", "control__icon-wrapper"))
                             <a href="@getReaderRevenueUrl(Membership, OldHeader)"
                             data-link-name="Register link"
-                            class="brand-bar__item--action brand-bar__item--split--first js-become-member js-change-become-member-link">
```

Comment:
Presumably `js-change-become-member-link` was only ever used for this test? Same for `newHeader.scala.html` below.

Reply:
Yes, I added for test4, then we used it for test 3. So I am deleting them 😄